java/src/main/java/lesson01/solution/Hello.java

```diff
@@ -32,8 +32,8 @@ public static void main(String[] args) {
             throw new IllegalArgumentException("Expecting one argument");
         }
         String helloTo = args[0];
-        Tracer tracer = Tracing.init("hello-world");
-        new Hello(tracer).sayHello(helloTo);
-        tracer.close();
+        try (Tracer tracer = Tracing.init("hello-world")) {
```

Comment:
is the tracer closable?

Reply:
Tracer is com.uber.jaeger.Tracer in this class, which implements the Closeable interface. io.opentracing.Tracer, however, does not extend the Closeable interface.